lib/rules/aria-roles.json

```diff
@@ -8,5 +8,5 @@
 	},
 	"all": [],
 	"any": [],
-	"none": ["invalidrole", "abstractrole", "unsupportedrole"]
+	"none": ["fallbackrole", "invalidrole", "abstractrole", "unsupportedrole"]
```

Comment:
You'll also have to update `invalidrole`, have it look at each token in a role attribute rather than the entire string. Otherwise that check will still return an issue for `role="spinbutton button"` for example. That should only fail `fallbackrole`.